.golangci.example.yml

```diff
@@ -228,6 +228,21 @@ linters-settings:
     # Allow declarations (var) to be cuddled.
     allow-cuddle-declarations: false
 
+  # The custom section can be used to define linter plugins to be loaded at runtime. See README doc
+  #  for more info.
+  custom:
+    # Each custom linter should have a name. This name should match with what the linter reports for itself.
+     example:
+      # The path to the plugin *.so. Required for each custom linter
+      path: /path/to/example.so
+      # If the linter should be enabled by default. Options in the `linters`
+      #  section override this, including `disable-all`. Optional, false by default.
+      enabled: true
```

Comment:
Why duplicate it with `linters` section? It looks difficult to understand for users.

Reply:
Hm. There are a couple of ways to reduce that duplication. The easiest and I think most consistent is to set the linter to enabled by default and remove this option. That does leave the 'problem' that if the user has `disable-all: true` in the linters section, that will override enable by default even for the custom linters. Thus, the user will be forced to define the linter again there. But that also preserves the ability to turn on and off custom linters on the command line.